swebench/harness/test_spec/test_spec.py

```diff
@@ -174,7 +183,7 @@ def make_test_spec(
     assert instance_image_tag is not None, "instance_image_tag cannot be None"
     instance_id = instance[KEY_INSTANCE_ID]
     repo = instance["repo"]
-    version = instance.get("version")
+    version = instance.get("version", "-1")
```

Comment:
I feel like "-1" might be more likely to break things than just an empty string

Reply:
Makes sense, `bool(version)` will then eval to `False`. Updated it.